plugins/destination/file/client/json.go

```diff
@@ -0,0 +1,62 @@
+package client
+
+import (
+	"bufio"
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+const maxJsonSize = 1024 * 1024 * 20
```

Comment:
This is only used for reading/scanner yet the name doesn't suggest that. (Could also move the const defn inside the read method)

Reply:
+1